apiserver/pkg/model/converter_test.go

```diff
@@ -126,24 +126,26 @@ var headSpecTest = rayv1api.HeadGroupSpec{
 
 var configMapWithoutTolerations = corev1.ConfigMap{
 	Data: map[string]string{
-		"cpu":             "4",
-		"gpu":             "0",
-		"gpu_accelerator": "",
-		"memory":          "8",
-		"name":            "head-node-template",
-		"namespace":       "max",
+		"cpu":                "4",
+		"gpu":                "0",
+		"gpu_accelerator":    "",
+		"memory":             "8",
+		"extended_resources": "{\"vpc.amazonaws.com/efa\": 32}",
```

Comment:
Consider "custom_resources" instead, which is more aligned to Ray terminology: https://docs.ray.io/en/latest/ray-core/scheduling/resources.html#custom-resources

Reply:
On second thought, maybe custom_resources is misleading because this is never passed into the `--resources` flag in `ray start`. Do you need to include this as a custom resoruce in Ray or is it enough to add it as a container resource?

Reply:
This is not for custom_resources, but the extended resource for kubernetes container.